Sources/Models/BLTNItem.swift

```diff
@@ -0,0 +1,152 @@
+     *
+     * Set this value to `false` to display the elements right away. If you set it to `true`,
+     * you'll need to call `manager?.hideActivityIndicator()` to show the UI.
+     */
+
+    @objc open var shouldStartWithActivityIndicator: Bool = false
+
+    /**
+     * Whether the item should move with the keyboard.
+     *
+     * You must set it to `true` if the item displays a text field. You can set it to `false` if you
+     * don't want the bulletin to move when system alerts containing a text field (ex: iTunes login)
+     * are displayed.
+     */
+
+    @objc open var shouldRespondToKeyboardChanges: Bool = true
+
+    /**
+     * The item to display after this one.
+     *
+     * If you set this value, you'll be able to call `manager?.displayNextItem()` to push the next item to
+     * the stack.
+     */
+
+    @objc(nextItem) open var next: BLTNItem?
+    
+    // MARK: - Event Handlers
+    
+    /**
+     * The block of code to execute when the bulletin item is presented. This is called after the
+     * bulletin is moved onto the view.
+     *
+     * - parameter item: The item that is being presented.
+     */
+
+        @objc open var presentationHandler: ((BLTNItem) -> Void)?
+
+    /**
+     * The block of code to execute when the bulletin item is dismissed. This is called when the bulletin
+     * is moved out of view.
+     *
+     * You can leave it `nil` if `isDismissable` is set to false.
+     */
+
+    @objc open var dismissalHandler: ((BLTNItem) -> Void)?
+
+    // MARK: - Interface
+
+    /**
+     * Creates the list of views to display inside the bulletin card.
+     *
+     * The views will be arranged vertically, in the order they are stored in the return array.
+     */
+
+    open func makeArrangedSubviews() -> [UIView] {
+        return []
+    }
+
+    /**
+     * Called by the manager when the item was added to the bulletin.
+     *
+     * Use this function to configure your managed views, and allocate any resources required
+     * for this item.
+     */
+
+    open func setUp() {
+        // no-op
+    }
+
+    /**
+     * Called by the manager when the item was removed from the bulletin.
+     *
+     * Use this function to remove any button target or gesture recognizers from your managed views, and
+     * deallocate any resources created for this item that are no longer needed.
+     */
+
+    open func tearDown() {
+        // no-op
+    }
+        
+    /**
+    * Called by the manager when bulletin item is about to be pushed onto the view.
+    */
+        
+    open func willDisplay() {
+        // no-op
+    }
+
+    /**
+     * Called by the manager when bulletin item is pushed onto the view.
+     */
+
+    open func onDisplay() {
+        presentationHandler?(self)
+    }
+
+    /**
+     * Called by the manager when bulletin item is dismissed. This is called after the bulletin
+     * is moved out of view.
+     */
+
+    open func onDismiss() {
+        presentationHandler?(self)
```

Comment:
This should be `dismissalHandler?(self)`, no?

Reply:
Good call! Thanks